lib/converterbase.rb

```diff
@@ -675,7 +675,7 @@ def jisage(line, num)
   end
 
   def border_symbol?(line)
-    @@symbols ||= open(File.join(Narou.get_preset_dir, "bordersymbols.txt"), "r:BOM|UTF-8") { |fp|
+    @@symbols ||= open(File.join(Narou.preset_dir, "bordersymbols.txt"), "r:BOM|UTF-8") { |fp|
```

Comment:
__[RuboCop]__

Security/Open: The use of `Kernel#open` is a serious security risk.


<!-- Comment by [Sider](https://sider.review) -->